python/lsst/meas/deblender/baseline.py

```diff
@@ -102,7 +101,7 @@ def __init__(self, footprint, maskedImages, psfs, psffwhms, log, filters=None,
             else:
                 avgNoise = [avgNoise]
         # Now check that all of the parameters have the same number of entries
-        if any([len(maskedImages)!=len(p) for p in [psfs, psffwhms, avgNoise]]):
+        if any([len(self.filters) !=len (p) for p in [psfs, psffwhms, avgNoise]]):
```

Comment:
Variable name `maskedImages` on line 108 doesn't exist. That raise will never work now.

Reply:
Also, did you mean to put the whitespace in this line *before* the `len` instead of after? :)